packages/docprovider/src/yprovider.ts

```diff
@@ -195,7 +195,7 @@ export class WebSocketProviderWithLocks
     }
     this._sendMessage(new Uint8Array([127]));
     // try to acquire lock in regular interval
-    const intervalID = setInterval(() => {
+    this._intervalID = setInterval(() => {
```

Comment:
You should make sure to clear the current `_intervalID` before creating a new one. Maybe we should also rename this to `this._requestLockInterval`

Reply:
I added a commit with the changes.